src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,156 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+import telemetry 0.30
+
+VPNTabNavigation {
+    property string _menuTitle: VPNl18n.TipsAndTricksSettingsEntryLabel
+    objectName: "settingsTipsAndTricksPage"
+
+    tabList: ListModel {
```

Comment:
Let's add `elide: Qt.ElideRight` around [here](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/733296c69bf6be9ade09eb4fddddbd388a796fc4/nebula/ui/components/VPNTabNavigation.qml#L71-L76) in VPNTabNavigation.qml to prevent very long strings from expanding beyond the horizontal confines of their tab.